ts/torch_handler/base_handler.py

```diff
@@ -69,37 +62,61 @@ def initialize(self, context):
         # Read the mapping file, index to object name
         mapping_file_path = os.path.join(model_dir, "index_to_name.json")
 
-        if os.path.isfile(mapping_file_path):
-            with open(mapping_file_path) as f:
-                self.mapping = json.load(f)
-        else:
-            logger.warning('Missing the index_to_name.json file. Inference output will not include class name.')
+        # Load class mapping for classifiers
+        mapping_file_path = os.path.join(model_dir, "index_to_name.json")
```

Comment:
duplicate line.

Reply:
Removed the duplicate. Moving it to the function seems a bit odd -- most methods in Python regarding files accept a full path instead of dir + filename.